index.html

```diff
@@ -237,7 +237,7 @@ <h3>
             <dd>
               <p>
 This family of specifications consists of documents that each define how to
-express proofs of integrity for verifiable credentials using a number of
+express and associate proofs of integrity with data, focusing on verifiable credentials and
 concrete serializations for each of the defined syntaxes. The specific set of
```

Comment:
```suggestion
express and associate proofs of integrity for verifiable credentials and concrete
serializations for each of the defined syntaxes. The Working Group would welcome
to see the usage of these techniques for data in general, but expressing those are
not in the scope of this deliverable. The specific set of
```

Reply:
I think this is good compromise.

Reply:
I prefer @brentzundel's base with @dlongley's modification below.